src/utils/ecnryption.ts

```diff
@@ -0,0 +1,12 @@
+import CryptoJS from 'crypto-js'
+const key = import.meta.env.VITE_SALT
```

Comment:
could this have a fallback string, please - so it works in environments where we haven't (yet) created the environment variable.